lib/readthis/serializers.rb

```diff
@@ -13,7 +13,7 @@ class Serializers
     }.freeze
 
     # The hard serializer limit, based on the number of possible values within
-    # a single 8bit integer.
+    # a single 3bit integer.
```

Comment:
Yes, important distinction between the number of bits and the integer it can represent.